i18n/fra/package.i18n.json

```diff
@@ -12,7 +12,7 @@
 	"cmake-tools.command.cmake.setVariantAll.title": "Sélectionner une variante pour tous les projets",
 	"cmake-tools.command.cmake.configure.title": "Configurer",
 	"cmake-tools.command.cmake.configureAll.title": "Configurer tous les projets",
-	"cmake-tools.command.cmake.build.title": "Version",
```

Comment:
This is currently "Build" -- not sure if that changes the translation.

Reply:
It shouldn't. "Build" can be translated in various ways here such as "Construire" or "Générer". I've just followed the terminology already used for the "buildAll" entry.